include/SDL3/SDL_storage.h

```diff
@@ -100,13 +106,31 @@ extern DECLSPEC SDL_Storage *SDLCALL SDL_OpenTitleStorage(const char *override,
  * \sa SDL_OpenStorage
  * \sa SDL_CloseStorage
  * \sa SDL_StorageReady
- * \sa SDL_StorageFileSize
- * \sa SDL_StorageReadFile
- * \sa SDL_StorageWriteFile
- * \sa SDL_StorageSpaceRemaining
+ * \sa SDL_GetStorageFileSize
+ * \sa SDL_ReadStorageFile
+ * \sa SDL_WriteStorageFile
+ * \sa SDL_GetStorageSpaceRemaining
  */
 extern DECLSPEC SDL_Storage *SDLCALL SDL_OpenUserStorage(const char *org, const char *app, SDL_PropertiesID props);
 
+/**
+ * Opens up a container for local filesystem storage.
```

Comment:
Definitely worth putting a giant warning that this should be for internal development only; the worst case scenario is that the engine gets coupled to this API for release builds, meaning they'll eventually have to port from SDL to... well, SDL.

Reply:
Do you want to draft a warning?

Reply:
Maybe something like:
"This is provided for development and tools. Portable applications should use SDL_OpenTitleStorage() for access to game data and SDL_OpenUserStorage() for user data."

Reply:
That works for me!